TimePicker/TimePickerBase.js

```diff
@@ -205,6 +207,7 @@ const TimePickerBase = kind({
 
 	styles: {
 		css,
+		dateTimeCss,
```

Comment:
This doesn't work. If you want this to work, it will need to be merged with the `css` object, but I'm reasonably certain this is not what you want to do here.